### ISSUE TITLE
q-to-bibliography-templates returns nothing

### ISSUE THREAD

**Issue by VladimirAlexiev** [fnielsen/scholia#564]
https://tools.wmflabs.org/scholia/q-to-bibliography-templates?q=Q33306866 returns only a comment "Generated with scholia.wikipedia q-to-bibliography-templates"

**Comment by fnielsen** [comment]
Yes, I see that the function or documentation (or both) is not well. 

Initially, I thought the tool to be used with an author, like:
https://tools.wmflabs.org/scholia/q-to-bibliography-templates?q=Q48975668

The "Select bibliography" was initially generated in this way. https://en.wikipedia.org/wiki/Robert_Kelly_(political_analyst)

It may make sense to extend the functionality. At least the documentation should be improved.

**Comment by mvolz** [comment]
This works now, but it doesn't include the authors.
